packages/terralib/lua/Layer.lua

```diff
@@ -22,6 +22,25 @@
 --
 -------------------------------------------------------------------------------------------
 
+local EncodingMapper = {
+	utf8 = "UTF-8",
+	cp1250 = "CP1250",
+	cp1251 = "CP1251",
+	cp1252 = "CP1252",
+	cp1253 = "CP1253",
+	cp1254 = "CP1254",
+	cp1257 = "CP1257",
+	latin1 = "LATIN1",  --Latin1 encoding (ISO8859-1)
+}
+
+local function isValidEncoding(encoding)
```

Comment:
This function could be simply 'return EncodingMapper[encoding]'. Could this be removed?